### ISSUE TITLE
[Spotlight] All Imagine Your Story Pages

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#106]
**URL:** https://spotlight.library.tamu.edu/spotlight/imagine-your-story
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.4.3 Contrast Minimum (AA)
**Issues:**  Contrast Error
**Notes:**  Local nav bar items don't have enough contrast when not selected
